tests/test_ocs_basic_install.py

```diff
@@ -34,19 +36,27 @@ def run(**kwargs):
     log.info("Generating install-config")
     # TODO: determine better place to create cluster directories - (log dir?)
     cluster_dir_parent = "/tmp"
-    base_name = test_data.get('cluster-name', 'ocs-ci-cluster')
+    cluster_name = test_data.get('cluster-name')
+    cluster_path = test_data.get('cluster-path')
     cid = random.randint(10000, 99999)
-    cluster_name = f'{base_name}-{cid}'
-
-    cluster_path = os.path.join(cluster_dir_parent, cluster_name)
+    if not (cluster_name and cluster_path):
+        cluster_name = f"{default.CLUSTER_NAME}-{cid}"
```

Comment:
Why is necessary to reset `cluster_name` when `cluster_path` is not defined? I'm just curious.

Reply:
good catch, I wanted yesterday to short this one but didn't realize this default name there. Thanks, will fix in next commit

Reply:
@mbukatov I hope I found the solution for this in next commit. Can you please take a look if it makes sense now? Thanks